app/views/dashboard/_supervisors_table.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if policy(:dashboard).see_supervisors_section? %>
```

Comment:
FYI, this partial was removed so you may run into conflicts once you pull down `new-layout`.